### ISSUE TITLE
Who should be the Copyright holders?

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTDESC/desc_package_template#2]
Over in https://github.com/DarkEnergyScienceCollaboration/CCL/issues/53 @dkirkby writes:

What does "copyright owned by the collaboration" mean?  That would be an exception to the general rule that *the creator of a work owns the copyright*, which might require a more formal agreement (see [here](http://fairuse.stanford.edu/overview/faqs/copyright-ownership/), for example).  For projects on github, I usually use something like (from [here](https://github.com/DarkEnergyScienceCollaboration/WeakLensingDeblending/blob/master/LICENSE)):

> Copyright (c) 2014 by the WeakLensingDeblending contributors on GitHub
> https://github.com/DarkEnergyScienceCollaboration/WeakLensingDeblending

This is consistent with the following advice from [here](http://fairuse.stanford.edu/overview/faqs/copyright-ownership/#who_owns_the_copyright_in_a_joint_work) and avoids having to manage a separate list of copyright holders:

> When two or more authors prepare a work with the intent to combine their contributions into 
> inseparable or interdependent parts, the work is considered joint work and the authors are 
> considered joint copyright owners. 

The contributors to a repo (the people who have had commits accepted) can be listed with, e.g.
```
% curl -s https://api.github.com/repos/DarkEnergyScienceCollaboration/CCL/stats/contributors \
| sed -n 's/ *"login": "\([a-z0-9]*\)",/\1/gp'
jmeyers314
alexabate
rmandelb
ismael2395
dkirkby
```
The contributors' contributions can be visualized at, e.g. https://github.com/DarkEnergyScienceCollaboration/CCL/graphs/contributors

Now, we did consult the SLAC lawyer (either directly or indirectly) about who the Copyright holder should be.  Can you please advise, @brianv0? I'm happy to implement David's above suggestion, with or without a weblink.

**Comment by drphilmarshall** [comment]
cc: @DarkEnergyScienceCollaboration/pubboard

**Comment by drphilmarshall** [comment]
To close the loop on this (and because CCL needed a license!) I implemented David's suggestions in PR #3 and asked @jchiang87 to review them, when he has a minute. The main thing I haven't done is  test that the template package works, which makes me think we need to get travis making a dummy package.
